synapse/lib/cell.py

```diff
@@ -1267,3 +1241,33 @@ async def sync(self):
         the writes that occurred on the leader before this call.
         '''
         return
+
+    async def ps(self, user):
+        isallowed = await self.isUserAllowed(user, ('task', 'get'))
+
+        retn = []
+        for task in self.boss.ps():
+            if (task.user.iden == user) or isallowed:
```

Comment:
we should either pass the user object or make it clear that we're passing in the user iden as a function arg because this totally threw me, thinking it was doing ``__eq__`` against the User object.